### ISSUE TITLE
Token error 'delete' on an old browser

### ISSUE THREAD

**Issue by mardon86** [jashkenas/backbone#4242]
Token error 'delete' at line 1621 in old android browser (Webkit <= 533)

```
var methodMap = {
    create: 'POST',
    update: 'PUT',
    patch: 'PATCH',
--> delete: 'DELETE',
    read: 'GET'
};
```
maybe we should put a quotation mark on delete key

**Comment by jgonggrijp** [comment]
Thanks for reporting @mardon86.

Pull requests welcome.

**Comment by EvanHahn** [comment]
It looks like this was fixed in 19ce298f789a60d55aaf3d12ab233656df9d564a and released in 1.4.1.

**Comment by jgonggrijp** [comment]
Thanks for making me aware of that @EvanHahn! Closing.
